ml-proto/host/params.ml

```diff
@@ -1,4 +1,4 @@
-let page_size = 4096L
+let page_size = 0x10000L (* 64 KiB *)
```

Comment:
Given that this is no longer a parameter, can you instead remove this definition completely? It should also vanish from the `host_params` type in the eval module.